MagicSpell/MacroFunction.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MacroFunction.swift
```

Comment:
I don't see any further code will be needed. In the event that we do need to add more helper functions, we can put them in another file if needed. "Functions.swift" tells me nothing about what this file does.